Example/Tests/Tests.swift

```diff
@@ -10,12 +11,12 @@ class Tests: XCTestCase {
 
     enum TestError: LocalizedError {
         case nullPointerException
+        case andersonsError
```

Comment:
Really?

Reply:
@sindhu238 would you rather it was named after you?